### ISSUE TITLE
VALIDATION FAILED for okay Dockerfile

### ISSUE THREAD

**Issue by cancan101** [Runnable/validate-dockerfile#13]
Not sure why I am getting this error:
```
$ docklint Dockerfile 
VALIDATION FAILED
Bad parameters
at line 1
```
Dockerfile:
```Dockerfile
FROM dockerfile/java:oracle-java6 
RUN curl -o XXX.jar YYYY.jar
CMD java -jar XXX.jar
EXPOSE 9324
```

**Comment by cflynn07** [comment]
Hey @cancan101,

I tested your sample Dockerfile and it didn't trigger a validation error
https://github.com/Runnable/validate-dockerfile/commit/e647d787bb84b8eb260d3a3fe06abc8b67cc3023

I noticed when copying/pasting your FROM line has a trailing whitespace character. This will trigger a validation error. I removed the trailing whitespace and the file validates.

**Comment by cancan101** [comment]
Shouldn't `validate-dockerfile` strip trailing whitespace before validating in that case?

**Comment by SomeKittens** [comment]
@cancan101 You are correct, this is a false positive.  Thanks for reporting!

**Comment by SomeKittens** [comment]
Fixed in `1.5.2`
